Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.rst

```diff
@@ -0,0 +1,113 @@
+It is typically used in emergency situations or as a safety measure. 
+It works by cutting off the electrical power to the various components of the telescope, 
+including the mount and dome, which stops them from being moved either by software commands or manually.
+
+The emergency stop can be triggered by users in certain circumstances, but it can also be activated automatically in the event of strong earthquakes 
+(currently not implemented), as well as in response to safety interlocks and software-based emergencies. 
+While the primary objective of the emergency e-stop procedure is to provide a quick and reliable means of halting the movement of the telescope and dome to avoid accidents and injuries, 
+it also serves the purpose of safeguarding the system.
+
+Below you will find a brief overview of the emergency stop procedure, covering:
+
+- The location of the relevant switches, 
+- How to activate the emergency stop, and
+- How to determine its current status.
+  
+
+.. _E-Stop-Procedure-Where-to-Find-It:
+
+Where to Find It
+================
+
+The emergency stop system in the Auxiliary Telescope can be triggered from three different locations. 
+One is located in the main ``AT Control Cabinet`` on the first floor. 
+A second one is located in the ``Dome Control Box`` on the second floor. 
+Both of these locations can be used to both engage and disengage the emergency stop.
+
+.. figure:: ./_static/e-stop-dome-control-box.jpeg
+    :name: e-stop-dome-control-box
+
+    Screenshot of the e-stop system located in the Dome Control Box.
+
+Additionally, a third emergency stop button is available upstairs in the control box used for manual movement of the dome components, 
+as illustrated in the accompanying figure. 
+It is worth noting, however, that this control box can only be used to initiate the emergency stop and cannot be used to release it, 
+as it does not have a ``Start`` button on it.
+
+.. figure:: ./_static/e-stop-up-stairs.jpeg
+    :name: e-stop-up-stairs
+
+    Screenshot of the e-stop system located upstairs.
+
+In the future, an emergency stop will also be added to the main telescope control room. 
+For now, the emergency e-stop system available in there will only work for the main telescope components.
+
+.. _E-Stop-Procedure-How-Engage-Disengage-It:
+
+How to Engage/Disengage It
+==========================
+
+**Engaging** the emergency stop system is a straightforward process that is both efficient and reliable. 
+To trigger it, simply **pull the Emergency Stop button**. To **release it**, simply **turn the red button clockwise** a quarter or so
+and then **pull the Start** button.
+
+.. figure:: ./_static/e-stop-buttons.jpeg
+    :name: e-stop-buttons
+
+    Screenshot the e-stop ``Emergency Stop`` and ``Start`` buttons.
+
+**Verifying** that the emergency stop system was properly engaged is also easy and straightforward. 
```

Comment:
Add to Ioana's suggestion, I would like to suggest that "Verifying that ..." can be changed to "How to Verify Proper Engagement of Emergency Stop" or etc. as a title of a new section (it is just my personal opinion, but "is also easy and straightforward" could be removed -- as you've put the photo so it already seems simple and straightforward.

Reply:
Thank you for you suggestion. I have created a new section for this. 

@ykangastro I prefer to say explicitly it is easy and straightforward so that people know that the procedure is that easy,  just to make sure that people is aware of that even though it is pretty obvious.